app/bundles/ReportBundle/Builder/MauticReportBuilder.php

```diff
@@ -371,7 +371,7 @@ private function applyFilters(array $filters, QueryBuilder $queryBuilder, array
     {
         $expr      = $queryBuilder->expr();
         $groups    = [];
-        $groupExpr = $queryBuilder->expr()->andX();
+        $groupExpr = $queryBuilder->expr()->and('');
```

Comment:
@escopecz this is driving me crazy, this statement should not exists, I can not find a solution for making any `CompositeExpression`, Do you have any?